### DISCUSSION TITLE
Bindable color

### DISCUSSION THREAD

**Discussion by adosikas** [zauberzeug/nicegui#1389] [Ideas / Feature Requests]
It would be nice if `BackgroundColorElement` and `TextColorElement` exposed the color as bindable property, so it can be used directly to bind to color inputs etc.

Good example are actually the examples for [color picker](https://nicegui.io/documentation/color_picker) and [color input](https://nicegui.io/documentation/color_input), which have to resort to `style()`.

**Comment by falkoschindler** [comment]
You're right, @adosikas. That would be a nice feature.

When the color value changes, e.g., from a CSS color to a Tailwind color, we would need to make sure to remove the CSS property before setting the Tailwind class. But it should be possible.

Would anyone like to create a pull request for this? See [CONTRIBUTING.md](https://github.com/zauberzeug/nicegui/blob/main/CONTRIBUTING.md#pull-requests) for more information. 🙂
